src/content/en/android/custom-tabs/headers/index.md

```diff
@@ -0,0 +1,178 @@
+project_path: /web/android/_project.yaml
+book_path: /web/android/_book.yaml
+description: Guide for adding HTTP CORS Headers in Custom Tab Intents.
+
+{# wf_published_on: 2020-08-12 #}
+{# wf_updated_on: 2020-08-12 #}
+{# wf_blink_components: N/A #}
+
+# HTTP Request Headers in Custom Tab Intents {: .page-title }
+
+## Introduction
+
+HTTP requests contain headers such as User-Agent or Content-Type. Apart from headers attached by browsers, Android apps may add extra headers, like Cookie or Referrer through the [`EXTRA_HEADERS`][1] Intent extra. For security reasons, Chrome filters some of the extra headers depending on how and where an intent is launched.
+
+[Cross-origin][2] requests require an additional layer of security as the client and server are not owned by the same party. This guide discusses launching such requests through Chrome [custom tabs][3]. Until Chrome 83, developers could add any headers when launching a Custom Tab. From version 83, Chrome started filtering all except [whitelisted][4] cross-origin headers, since non-whitelisted headers posed a security risk. Starting with Chrome 86, it is possible to attach non-whitelisted headers to cross-origin requests, when the server and client are related using a [digital asset link][5]. This behaviour is summarised in [Table 1](#table-1).
```

Comment:
I'm wondering if it's clear for developers that an Android app opening an Intent to open a URL in the browser is also a cross-origin request. Should we call this out explicitly here?

Reply:
We're doing an effort to avoid potentially offensive terms to our docs as well. "whitelist" is one of those terms. Maybe we can replace those with "safelist" or "allowlist"?